server/aws-lsp-codewhisperer/src/language-server/chat/constants.ts

```diff
@@ -31,6 +31,7 @@ export const HELP_MESSAGE = `I'm Amazon Q, a generative AI assistant. Learn more
 
 export const DEFAULT_HELP_FOLLOW_UP_PROMPT = 'How can Amazon Q help me?'
 
+// TO BE DEPRECATED, use DEFAULT_EXCLUDE_DIRS and DEFAULT_EXCLUDE_FILES instead
```

Comment:
AFAIK this is only used by fileSearch. Is there any reason file search can't use the tree output utility as well?

If so, we could remove the `readDirectoryRecursively` completely in favor of the tree format.

Reply:
fileSearch is pending major changes, so leave it as it is for now